strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/io/ArtifactOutputStreamTest.java

```diff
@@ -107,26 +74,26 @@ public void testCreateWithTemporaryLocation(TestInfo testInfo)
 
         assertTrue(Files.exists(artifactPathTemp), "Failed to create temporary artifact file!");
 
-
-
         artifactPathTemp.getFileSystem().provider().moveFromTemporaryDirectory(artifactPathTemp);
 
         assertTrue(Files.exists(artifactPath), "Failed to the move temporary artifact file to original location!");
     }
 
     @Test
-    public void testCreateWithTemporaryLocationNoMoveOnClose(TestInfo testInfo)
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
+    public void testCreateWithTemporaryLocationNoMoveOnClose(@TestRepository(repository = "aost=tcwtlnmoc-releases",
+                                                                             layout = Maven2LayoutProvider.ALIAS)
+                                                             Repository repository,
+                                                             @MavenTestArtifact(id = "org.carlspring.foo:temp-file-test",
+                                                                                versions = { "1.2.4" })
+                                                             Path path)
             throws IOException
     {
-        final Storage storage = getConfiguration().getStorage(STORAGE0);
-        final Repository repository = storage.getRepository(getRepositoryName(testInfo));
-
         final Artifact artifact = ArtifactUtils.getArtifactFromGAVTC("org.carlspring.foo:temp-file-test:1.2.4:jar");
         final ArtifactCoordinates coordinates = new MavenArtifactCoordinates(artifact);
 
         RepositoryPath artifactPath = repositoryPathResolver.resolve(repository, coordinates);
```

Comment:
you don't need to resolve it, just use `RepositoryPath repositoryPath = (RepositoryPath)path.normalize()`

Reply:
Right, I see now! Thanks for this example!